utils/prezly/api/PrezlyApi.ts

```diff
@@ -31,6 +33,28 @@ export default class PrezlyApi {
         return this.newsroom;
     }
 
+    async getNewsroomLanguages() {
+        if (!this.newsroomLanguages) {
+            this.newsroomLanguages = (
+                await this.sdk.newsroomLanguages.list(this.newsroomUuid)
+            ).languages;
+        }
```

Comment:
Does this kind of caching make sense? API client is anyway created with each request, so that this data will be fetched again.
Additionally, this caching is not done right, because if `getNewsroomLanguages` is called twice in a row, it will make two requests.
More proper way of doing this would be
```
    private newsroomLanguagesPromise?: Promise<NewsroomLanguageSettings[]>;
    ...
    async getNewsroomLanguages() {
        if (this.newsroomLanguagesPromise) {
           return this.newsroomLanguagesPromise;
        }

        const { languages } =  await this.sdk.newsroomLanguages.list(this.newsroomUuid);
        this.newsroomLanguagesPromise = Promise.resolve(languages);

        return languages;
    }
```

Reply:
you're right, I thought it made sense for development but it doesn't, the api is built anyway